walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,46 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
+
+  /**
+   * Given the meeting information,
+   * finds the times when the meeting could happen that day.
+   */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    ArrayList<TimeRange> busyTimes = new ArrayList<>();
+    for (Event event : events) {
+      for (String attendee : request.getAttendees()) {
+        if (event.getAttendees().contains(attendee)) {
+          busyTimes.add(event.getWhen());
+          break;
+        }
+      }
+    }
```

Comment:
Can you add a comment explaining what you're trying to do here? Similar to the for loop below.